### PR TITLE
Fix some strings and translation stuff

### PR DESCRIPTION
**Pull request by LorenDB**
This was recently discussed in #nheko:nheko.im﻿

### CODE REVIEW

File: resources/qml/NotificationWarning.qml [hunk 1]
```diff
@@ -29,7 +29,7 @@ Item {
             anchors.rightMargin: 10
             anchors.bottom: parent.bottom
             color: Nheko.theme.red
-            text: qsTr("You will be pinging the whole room")
+            text: qsTr("You are about to ping the whole room")
```
Comment (deepbluev7) [inline resources/qml/NotificationWarning.qml:32]: I don't think that is much of an improvement. "ping" is not really an understandable word? Maybe "You will be causing a notification for everyone in the room"?

Comment (LorenDB) [reply resources/qml/NotificationWarning.qml:32]: Or "you are about to notify the whole room"?

Comment (deepbluev7) [reply resources/qml/NotificationWarning.qml:32]: Sounds weird, but my English isn't great :D

Comment (LorenDB) [reply resources/qml/NotificationWarning.qml:32]: Well, "will be causing" is not really how Americans talk ;)

Comment (deepbluev7) [reply resources/qml/NotificationWarning.qml]: I guess, but I wanted to emphasize, that you are responsible for a lot of pings. I guess this is fine though, thank you!

**Pull request merged**